zeebe/atomix/cluster/src/test/resources/log4j2-test.xml

```diff
@@ -3,7 +3,7 @@
 
   <Appenders>
     <Console name="Console" target="SYSTEM_OUT">
-      <PatternLayout pattern="%d{HH:mm:ss.SSS} [%X{actor-name}] %-5level %logger{36} - %msg%n"/>
+      <PatternLayout pattern="%d{HH:mm:ss.SSS} [%X{actor-name} %X{actor-scheduler}] %-5level %logger{36} - %msg%n"/>
```

Comment:
🔧 We could also call it node-id, would make more sense I think.

Reply:
I'm using `actor-scheduler` because it's the same used in "production files"